regalii/swagger_documentation
# Modify introductory text of Merchant API

#### What's this PR do?
Adds changes from the following document: https://docs.google.com/document/d/1pvu_etjIrpo7F6S54lcbogTiYibxzd36z6mlZa3uV2Y/edit

#### How should this be manually tested?
Copy and paste apix_merchant_protocol.yaml into a new project in SwaggerHub to see the output.

#### Questions:
- Where should images required be stored?
